analysis/dataset_definition_med_status_data_development.py

```diff
@@ -0,0 +1,124 @@
+from ehrql import create_dataset, months, days
+from ehrql.tables.tpp import patients, clinical_events
+from ehrql.tables.raw.tpp import medications
+
+from datetime import date
+
+from codelists import *
+
+dataset = create_dataset()
+dataset.define_population(patients.exists_for_patient())
+
+# Pharmacy first clinical codes
+pharmacy_first_event_codes = [
+    "1659111000000107",
+    "1659121000000101",
+    "1577041000000109",
+    "983341000000102",
+]
+
+# Combine all medication codelists
+pharmacy_first_med_codes = (
+    acute_otitis_media_cod
+    + first_impetigo_treatment_cod
```

Comment:
Remove "first_" for consistency. See also change to codelists.py
```suggestion
    + impetigo_treatment_cod
```

Reply:
thanks :) addressed in ca1e1573c933a290905a92e8d8cc6065fe151a77